src/config.ts

```diff
@@ -53,6 +53,12 @@ export const config = configOrder({
     type: "boolean",
     default: true,
   },
+  ligatures: {
+    title: "Ligatures",
+    description: "Enables ligatures support (experimental).",
```

Comment:
ligatures addon is not experimental

Reply:
Again what is the benefit of this checkbox?

Reply:
The fact that it does not work with WebGL makes it experimental 🤔. That's what I meant.

> Again what is the benefit of this checkbox?

It is mostly for consistency with the other two. But I can of think two use cases:
- If someone sees that it makes a problem for their terminal, they can disable it. We cannot predict everyone's workflow.
- If someone does not use ligatures, they can speed up the loading of the Terminal a bit (since we use dynamic import).

Reply:
> The fact that it does not work with WebGL makes it experimental

No, that is what makes webgl experimental. xTerm even says that on the webgl addon.

> If someone sees that it makes a problem for their terminal, they can disable it.

If this is a goal than we need a lot more checkboxes for other features just incase they break.

> If someone does not use ligatures, they can speed up the loading of the Terminal a bit (since we use dynamic import).

I can understand this, but I don't think the few milliseconds this saves is worth people's time of having to uncheck the box.